lib/cryptoexchange/exchanges/coinhouse/services/pairs.rb

```diff
@@ -0,0 +1,36 @@
+module Cryptoexchange::Exchanges
+  module Coinhouse
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Coinhouse::Market::API_URL}/tickers"
+        TARGET = 'BTC'
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          market_pairs = []
+          output.each_key do |pair|
+            base, target = strip_pairs(pair)
+            next unless base && target
+            market_pairs << Cryptoexchange::Models::MarketPair.new(
+              base: base,
+              target: target,
+              market: Coinhouse::Market::NAME
+            )
+          end
+          market_pairs
+        end
+
+        def strip_pairs(pair_symbol)
+          target = pair_symbol[-3..-1]
```

Comment:
@Swingcloud  this assumes that the symbol is always 3 character if im not wrong.
https://coinhouse.eu/v2/tickers returns symbols like doge, dash, etc.
I think check the binance example, we have a hack to detect that, sort of..

Reply:
ok let me change it